library/std/src/random.rs

```diff
@@ -0,0 +1,106 @@
+//! Random value generation.
+//!
+//! The [`Random`] trait allows generating a random value for a type using a
+//! given [`RandomSource`].
+
+#[unstable(feature = "random", issue = "none")]
+pub use core::random::*;
+
+use crate::sys::random as sys;
+
+/// The default random source.
+///
+/// This asks the system for the best random data it can provide, meaning the
+/// resulting bytes *should* be usable for cryptographic purposes. Check your
```

Comment:
Seems like a problem to say _should_ here because then its completely unreliable. imo std should guarantee this or not.

Reply:
Yeah, makes sense. Its outside of std's control to enable all the various kernel configs etc.